apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.csproj

```diff
@@ -36,6 +36,12 @@
     <Reference Include="System.Transactions" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Update="SpannerCommand.cs">
```

Comment:
Oh the joy of these appearing and disappearing at will. Let's hope they just stay here...